app/src/main/java/com/example/cr554/newsapp/Util/APIKey.java

```diff
@@ -0,0 +1,9 @@
+package com.example.cr554.newsapp.Util;
+
+/**
+ * Created by cr554 on 2/22/2017.
+ */
+
+public class APIKey {
+    public static final String FLICKER_API_KEY ="e6678f97-7cd9-4156-88f7-f5f508c29ed3";
```

Comment:
as noted above ^^. My tutorial also didn't comply to the guidelines, especially since it was such a small project. It's not a big deal for this one, but it might be a big deal in the future so I just want to make you aware of it ^^